code/iaas/service-discovery/api/src/main/java/io/cattle/platform/compose/export/impl/ComposeExportServiceImpl.java

```diff
@@ -506,6 +507,9 @@ private void translateV1VolumesToV2(Map<String, Object> cattleServiceData,
         if (StringUtils.isEmpty(volumeDriver)) {
             return;
         }
+        if ("null".equals(volumeDriver)) {
```

Comment:
wrong fix. The correct fix is to not use String.valudOf() above. That is returning "null" when the value is null.